packages/theme/src/cli/utilities/theme-environment/types.ts

```diff
@@ -52,6 +52,16 @@ export interface DevServerContext {
    * theme.
    */
   themeEditorSync: boolean
+
+  /**
+   * Additional options for the development server.
+   */
+  options?: {
```

Comment:
I believe we will make this non-optional (just leaving a reminder from our pairing) ☺️